### ISSUE TITLE
Most examples throw `TypeError: Cannot read property 'isPlaying' of null` in Chrome (native custom elements)

### ISSUE THREAD

**Issue by cvan** [aframevr/aframe#1800]
**Description:**

[http://localhost:9000/examples/showcase/anime-UI/](http://localhost:9000/examples/showcase/anime-UI/)
![image](https://cloud.githubusercontent.com/assets/203725/17686335/8ec121b4-6320-11e6-8686-f95ea6e4d5e2.png)

[http://localhost:9000/examples/animation/aframe-logo/](http://localhost:9000/examples/animation/aframe-logo/)
![image](https://cloud.githubusercontent.com/assets/203725/17686354/ae6ef3a6-6320-11e6-88e0-e6c655e80c23.png)

```js
a-entity.js:258Uncaught (in promise) TypeError: Cannot read property 'isPlaying' of null(…)entityLoadCallback @ a-entity.js:258emitLoaded @ a-node.js:103
a-entity.js:258Uncaught (in promise) TypeError: Cannot read property 'isPlaying' of null(…)entityLoadCallback @ a-entity.js:258emitLoaded @ a-node.js:103
localhost/:1 [.Offscreen-For-WebGL-0x7fb7eb86e000]RENDER WARNING: Render count or primcount is 0.
localhost/:1 [.Offscreen-For-WebGL-0x7fb7eb86e000]RENDER WARNING: there is no texture bound to the unit 0
```

- A-Frame Version: `115cecc`
- Platform/Device: Google Chrome Version 52.0.2743.116 (64-bit); Mac OS X

**Comment by ngokevin** [comment]
I'm not getting this on the same A-Frame version/browser/device, can you restart your server to make sure, if you haven't already? It should be fixed in e9c27cc91efca52ac0bcf329101144ca573a180b

**Comment by cvan** [comment]
Hmm, I had opened it in an incognito window too. Oddly but gladly, I can't reproduce it now. Will close. Thanks.
